From 7186a4dfdfaf707ca9344fca2abada1f1d3d2a0d Mon Sep 17 00:00:00 2001
From: Yongxin Liu <yongxin.liu@windriver.com>
Date: Wed, 11 Jul 2018 06:50:58 +0000
Subject: [PATCH] tpm: remove __initdata before up2_board_dmi_table

Wrongly using __initdata causes kernel section mismatch warning.
Because tpm_chip_alloc is not a initialization function.

Signed-off-by: Yongxin Liu <yongxin.liu@windriver.com>
---
 drivers/char/tpm/tpm-chip.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/char/tpm/tpm-chip.c b/drivers/char/tpm/tpm-chip.c
index f72474c..22d743d 100644
--- a/drivers/char/tpm/tpm-chip.c
+++ b/drivers/char/tpm/tpm-chip.c
@@ -175,7 +175,7 @@ static int tpm_class_shutdown(struct device *dev)
 	return 0;
 }
 
-static const struct dmi_system_id __initdata up2_board_dmi_table[] = {
+static const struct dmi_system_id up2_board_dmi_table[] = {
 	{
 		.matches = {
 			DMI_MATCH(DMI_BIOS_VENDOR, "American Megatrends Inc."),
-- 
1.7.9.5

